firefox_puppeteer/ui/toolbars.py

```diff
@@ -315,6 +315,22 @@ def is_open(self):
         return self.popup.get_attribute('state') == 'open'
 
     @property
+    def is_complete(self):
+        """Returns when this popup is open and search results are complete.
+
+        :returns: True when the popup is open including any search results, otherwise false.
+        """
+        return self.marionette.execute_script("""
+          Components.utils.import("resource://gre/modules/Services.jsm");
+          let win = Services.focus.activeWindow;
```

Comment:
You should pass-in the browser window into the `execute_script` call. Otherwise we have an inconsistency between the Marionette known active window and another focused one.